### ISSUE TITLE
No idea how to reset the draw now

### ISSUE THREAD

**Issue by girkers** [home-assistant/home-assistant.github.io#18076]
### Feedback

In previous versions of this implementation when it was done by hand, all of the buttons where set through the configuration YAML which was simple enough. Now that the integration is configured through the UI there is no SIMPLE way to configure the core function of Resetting the Waste draw.

In the documentation you mention that you can now add this as a Service, but I have no idea where to put this. Looking at https://www.home-assistant.io/docs/scripts/service-calls/ still does not tell me where to put the code you mention, it also links to https://www.home-assistant.io/integrations/homeassistant#services but again NO where does it show me where to put the code.

I first have to ask why the CORE functionality is NO longer available in the base UI install?

With documentation as to the reporting of this issue, I have absolute NO idea how to implement this action without further information/instruction, even a link to documentation on how to implement this, please I am totally lost.

### URL

https://www.home-assistant.io/integrations/litterrobot/

### Version

2021.6.1

**Comment by frenck** [comment]
You can use the service call described here:

<https://www.home-assistant.io/integrations/litterrobot/#reset_waste_drawer>

In your automation actions or script sequences in YAML.
Additionally, those are also available via the UI, when creating an automation or script, the service is selectable when the integration is activated for your instance.

If you want to manual run/call the service, you can use the developer tools:

[![Open your Home Assistant instance and show your service developer tools with a specific service selected.](https://my.home-assistant.io/badges/developer_call_service.svg)](https://my.home-assistant.io/redirect/developer_call_service/?service=litterrobot.reset_waste_drawer)

**Comment by girkers** [comment]
Thanks for the reply but this doesn't help me understand actually how to do it. I appreciate there is documentation to do what you are saying but I just don't get HOW to actually implement it.

Think of it this way, I have HA installed and the first device I add is my litter robot, I have never done anything that resembles using services/automation, the code quoted means absolutely nothing to me. How do I use a "Service"? Where do I put that code quoted in the link you quoted?

I have found this: https://www.home-assistant.io/docs/scripts/service-calls/ and looked at the developer tools as per other documentation I have looked and the service is not listed. When I follow your button, which is awesome by the way, and click Call Service, it fails with this:

![image](https://user-images.githubusercontent.com/66195700/121128326-91065080-c86e-11eb-9bee-7b6388797ed2.png)

In the whole of this process I have found how to call service  from a button card, but with the service is missing.

Wait I just "reload" the integration and I found that I have a service called "Script: Reset Litter-Robot waste drawer"

![image](https://user-images.githubusercontent.com/66195700/121128971-96b06600-c86f-11eb-8f9b-65d8e1d914c4.png)

So it looks like I can reset the draw, however the documentation is still a little confusing.
